tests/conftest.py

```diff
@@ -22,29 +19,32 @@ def pytest_generate_tests(metafunc):
     if 'register_domain' in metafunc.fixturenames:
         metafunc.parametrize("register_domain", metafunc.config.option.register_domain)    
 
+def _entry_schema_properties():
+    return {
+        "entry-number": {
+            "type": "string",
+            "pattern": "^\d+$"
+        },
+        "item-hash": {
+            "type": "string",
+            "pattern": "^sha-256:[a-f\d]{64}$"
+        },
+        "entry-timestamp": {
+            "type": "string",
+            "pattern": "^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$"
```

Comment:
@chr1sdi suggested that we should extract these as (say) a `timestamp_datatype` thing because it's repeated all over the place